test/unified_format.py

```diff
@@ -511,6 +515,25 @@ def match_event(self, expectation, actual):
                 'Unsupported event type %s' % (event_type,))
 
 
+def coerce_result(opname, result):
```

Comment:
Should this be a method of `MatchEvaluatorUtil` class?

Reply:
I see no benefit to move it there.